includes/Classifai/Watson/Linker.php

```diff
@@ -71,11 +85,13 @@ public function link( $post_id, $output, $options = [] ) {
 	 *
 	 * Eg:- /animals/pets/cats
 	 *
-	 * @param int   $post_id The id of the post to link
+	 * @param int   $post_id The id of the post to link.
 	 * @param array $categories The list of categories to link
-	 * @return void
+	 * @param bool  $link_terms Whether link terms to post or return terms.
+	 *
+	 * @return array|\WP_Error List of the terms to link. WP_Error class object on error.
 	 */
-	public function link_categories( $post_id, $categories ) {
+	public function link_categories( int $post_id, array $categories, bool $link_terms = false ) {
```

Comment:
Sorry, not to be super nit picky but thinking that 1) because these are existing methods that would link terms by default and 2) the names of these methods are `link_*`, I think we should keep them linking terms by default. So `$link_terms = true` and if you don't want that functionality, you'd pass in `false` there